tiled_layers/export-vector-tiles/README.md

```diff
@@ -34,7 +34,7 @@ When the vector tiled layer loads, zoom in to the extent you want to export. The
 
 **NOTE:** Downloading Tiles for offline use requires authentication with the web map's server. To use this sample, you will require an [ArcGIS Online](https://www.arcgis.com) account.
 
-Vector tiles have high drawing performance and smaller file size compared to regular tiled layers, due to consisting solely of points, lines, and polygons. However, in ArcGIS Runtime SDK they cannot be displayed in scenes. Visit the [ArcGiS Online Developer's portal](https://developers.arcgis.com/java/latest/guide/layer-types-described.htm#ESRI_SECTION1_0A26749D5D094DAAA9DC12B2F9559E9E) to learn more about the characteristics of ArcGIS vector tiled layers.
+Vector tiles have high drawing performance and smaller file size compared to regular tiled layers, due to consisting solely of points, lines, and polygons. However, in ArcGIS Runtime SDK they cannot be displayed in scenes.
```

Comment:
@Rachael-E Just to check this line - is it staying as "ArcGIS Runtime SDK", but reference to Java are changing to API?

Reply:
oh nice find: this should be API too, I'll update